### ISSUE TITLE
Creating a buffer/delay to displayed Segments

### ISSUE THREAD

**Issue by xSlikZodiac** [collabora/WhisperLive#270]
Hey there, im new to github so if im creating this in the wrong area, please dont blast me. I've noticed with whisper models and implementations like this one that the displayed text segments often "self correct" or even jitter. this Jitter is even depicted in the main Demo during the chrome extension video demo. My thought was in the client, is there anyway to add a buffer to the words? maybe a 1-2 word buffer so it gives time for this? I know the alternative is waiting for the segment to complete but that would defeat the purpose of it being "Live" since it wouldn't be as real time. Im saying this because if youre trying to read along and the text is constantly updating words, punctuation, symbols, etc it makes it incredibly hard to follow. So im curious to see if anyone has found a way to prevent this while maintaining a Live Aspect. I know Youtube LiveStream captions does this well but even they have a 30 second delay. im curious to see if anyone has a work around this that is still "Live" but accurately displayed. 

Thanks!!

**Comment by AdolfVonKleist** [comment]
What you're asking for is basically impossible.  The reason it changes it because the early hypotheses are less reliable and lower confidence.  You either have to wait - probably 5s is enough IMO - or accept that it is going to change.  Even people behave this way; I'll often revise my 'internal recognition' some seconds afterwards.
